CODEOWNERS

```diff
@@ -55,12 +56,9 @@
 /xwf/ @magma/approvers-xwf
 /feg/radius @magma/approvers-xwf @magma/approvers-cwf
 
-/src/go/ @markjen
-
```

Comment:
@markjen seems to have left the project in November 2021.

Reply:
maybe make that part of @magma/approvers-orc8r instead of dropping it?

Reply:
Yes. That was [my view as well](https://github.com/magma/magma/pull/12576/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR42). I moved it to be consistent with the grouping.